Libplanet/Store/IStore.cs

```diff
@@ -244,5 +244,16 @@ void PutBlock<T>(Block<T> block)
         long CountTransactions();
 
         long CountBlocks();
+
+        /// <summary>
+        /// Forks transaction nonces from
+        /// <paramref name="sourceChainId"/> to
+        /// <paramref name="destinationChainId"/>.
+        /// </summary>
+        /// <param name="sourceChainId">The chain ID of transaction nonces to
+        /// fork.</param>
+        /// <param name="destinationChainId">The chain ID of destination
+        /// transaction nonces.</param>
```

Comment:
How about referencing words to its meaning code? I don't think my suggestion is the best way but I think it will be better to reference like my suggestion does.

```suggestion
        /// <summary>
        /// Forks <see cref="Transaction{T}"/> <see cref="Transaction{T}.Nonce"/>s from
        /// <paramref name="sourceChainId"/> to
        /// <paramref name="destinationChainId"/>.
        /// </summary>
        /// <param name="sourceChainId">The chain <see cref="BlockChain{T}.Id"/> of
        /// <see cref="Transaction{T}"/> <see cref="Transaction{T}.Nonce"/>s to fork.</param>
        /// <param name="destinationChainId">The chain <see cref="BlockChain{T}.Id"/> of destination
        /// <see cref="Transaction{T}"/> <see cref="Transaction{T}.Nonce"/>s.</param>
```